packages/story-editor/src/components/canvas/utils/useFullbleedMediaAsBackground.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import isTargetCoveringContainer from '../../../utils/isTargetCoveringContainer';
+import { useStory, useCanvas } from '../../../app';
+
+function useFullbleedMediaAsBackground({ selectedElement }) {
+  const { setBackgroundElement, currentPage } = useStory((state) => ({
+    setBackgroundElement: state.actions.setBackgroundElement,
+    currentPage: state.state.currentPage,
```

Comment:
Since we're only using the `isDefaultBackground`, we could get that directly instead of getting the whole `currentPage`, e.g. 
```
const { setBackgroundElement, isBackgroundElement } = useStory((state) => ({
    setBackgroundElement: state.actions.setBackgroundElement,
    isBackgroundElement: state.state.currentPage?.elements[0]?.isDefaultBackground,
...
```

If I'm not mistaken then @littlemilkstudio has done quite a bit of performance work to improve how we're using the selectors and reduce re-rendering, and that's one part of it -- whenever possible, to select precisely what we need instead of the whole object (including having some logic inside the selectors, example):
https://github.com/GoogleForCreators/web-stories-wp/blob/597789a61e47ee739038e62d3234de7485ee09cb/packages/story-editor/src/components/canvas/selection.js#L30-L34

 (@littlemilkstudio to clarify / correct me if I'm lying about something here 😛 )

Reply:
I was wondering if there will be any gain here, you know, re-rendering parents would wipe out those gains, e.g. singleSelectionMoveable is re-rendering, it looks like there is no gain, but I will do that in case I'm wrong. Maybe Max will shed some light here.

Reply:
Even if there's no gain specifically here, it might still be better to use this as a new best practice everywhere -- probably no point getting the whole object if we only need one property from it, especially if in some places it will be beneficial for preventing re-rendering as well. Thoughts?